cypher/cypher-docs/src/docs/dev/ql/administration/security/user-management-syntax-alter-user.asciidoc

```diff
@@ -1,10 +0,0 @@
-.Command syntax
```

Comment:
neo4j-documentation/4.3/cypher/cypher-docs/target/docs/dev/ql/administration/security/administration-security-users-and-roles.adoc